### ISSUE TITLE
invisible mesh

### ISSUE THREAD

**Issue by shuzzl** [SeriousCache/UABE#555]
After I replace the mesh (import raw), it becomes invisible in the game. The model has weights set and is attached to the armature. Unity game version and my unity version are the same

**Comment by wycsbg** [comment]
Have you solved it? I have the same problem.

**Comment by wycsbg** [comment]
> After I replace the mesh (import raw), it becomes invisible in the game. The model has weights set and is attached to the armature. Unity game version and my unity version are the same

Have you solved it? I have the same problem.

**Comment by shuzzl** [comment]
> > After I replace the mesh (import raw), it becomes invisible in the game. The model has weights set and is attached to the armature. Unity game version and my unity version are the same
> 
> Have you solved it? I have the same problem.

idk, I haven't found the answer to that question. Have you already been able to solve the problem?

**Comment by wycsbg** [comment]
> > > After I replace the mesh (import raw), it becomes invisible in the game. The model has weights set and is attached to the armature. Unity game version and my unity version are the same
> > 
> > 
> > Have you solved it? I have the same problem.
> 
> idk, I haven't found the answer to that question. Have you already been able to solve the problem?

Unfortunately, I haven't been able to solve this problem yet

**Comment by UsernameBillion** [comment]
Same issue here

**Comment by ZahidRiefqi** [comment]
same here, when I tried to import the model to unity game it didnt show anything at all

**Comment by BilawalLaghari** [comment]
Also I have same issue

**Comment by Pirangunter21** [comment]
Turns out the reason why mesh models are invisible is because the original mesh are skinned mesh with mapped avatar (replacing  this with custom mesh would break the mesh because it has no same avatar as the original). To replace this you need to decompile the game data to unity asset and re-convert the file (was prefab) to .fbx  in unity game engine with the same version as the game. That way you can replace skinned mesh properly. 

tl;dr you need to use asset ripper & asset extractor to convert the file to .asset package then convert desired prefab parented to skinned mesh renderer to .fbx then you can replace the model with custom one.

**Comment by AiglentinaA** [comment]
I have the same issue, but...reverse, I guess? I'm using UABE 3.0 Beta 1, and the unity is 2020.3.13f1 (same as the game). The object turns huge and freaks out when I look at it. I've tried about 5 times, all slightly different ways. The issue is the same though.
I see Pirangunter21 found the problem, but I don't exactly understand it due to lack of knowledge...Thank you Pirangunter21 though.
